More clearly indicate positivity of Math.abs return value (#21532)
mdn/content: files/en-us/web/javascript/reference/global_objects/math/abs/index.md

### Return value

> The absolute value of `x`. If `x` is negative (including `-0`), returns `-x`. Otherwise, returns `x`. The result is therefore always a non-negative number that's not `-0`.

Reviewer: Can you say:

```suggestion
The absolute value of `x`. If `x` is negative (including `-0`), returns `-x`. Otherwise, returns `x`. The result is therefore always a positive number or `0`.
```

...to have fewer negatives in that sentence (I count three)?

Author: Ah, looks good. Technically I could just say "non-negative", since -0 is negative... This is definitely stuff you would not worry in math.
